src/entities/ChatName.js

```diff
@@ -0,0 +1,27 @@
+const { Table, Entity } = require('dynamodb-toolbox')
+const { DocumentClient } = require('../../config/db')
+
+const ChatNameTable = new Table({
+  name: 'chatNames',
+  partitionKey: 'chatName',
+  sortKey: 'chatId',
+  DocumentClient,
+})
+
+const ChatName = new Entity({
```

Comment:
@georgeijidola if ChatNameTable is intended to be for multiple room support, please rename this to ChatRoom so it's more clear/familiar.

Reply:
actually I figured I don't need ChatName anymore